src/Features/LanguageServer/Protocol/Handler/FoldingRanges/FoldingRangesHandler.cs

```diff
@@ -1,16 +1,21 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Composition;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Structure;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
-namespace Microsoft.CodeAnalysis.LanguageServer
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
-    internal static class FoldingRangesHandler
+    [Shared]
+    // TODO - Use method name from VS language server package once updated.
```

Comment:
make work item? 

asking to open work item since there will be a lot of clean up we are thinking. doesn't need to create new item per issue. we can have 1 work item with bunch of checkbox.

Reply:
Good catch.. this should be updated to the actual method name since it's now updated :)

I've been created individual work items on the project since they are very lightweight ( I can just create a new card really fast) - It won't clutter up the existing issues either.
See https://github.com/dotnet/roslyn/projects/45